### ISSUE TITLE
Suggestion: Enable for server-side only

### ISSUE THREAD

**Issue by CplPibald** [sargunv/always-drop-loot#1]
Great and simple mod idea.  Thanks!

Can you please enable this for server-side only, so it doesn't need to be installed on clients?  This allows it to be dropped in to servers without changing the mod packs.  You just need to add `acceptableRemoteVersions` to the @Mod annotation.

```
@Mod(modid = AlwaysDropLootMod.MOD_ID, useMetadata = true, acceptableRemoteVersions = "*")
public class AlwaysDropLootMod {
```

**Comment by sargunv** [comment]
Ah, nice catch. Will take care of that shortly.

**Comment by sargunv** [comment]
Fixed in 900887851522a970ee39ef8dfeceab046863bff1

**Comment by sargunv** [comment]
Released in [version 1.0.1](https://github.com/sargunv/always-drop-loot/releases/tag/1.0.1)
